ext/storage_sources/local_store/local_store.c

```diff
@@ -235,7 +236,7 @@ local_config_dup(LOCAL_STORAGE *local, WT_SESSION *session, WT_CONFIG_ITEM *v, c
         free(p);
         return (ret);
     }
-    (void)strncat(p, suffix, len);
+    (void)strncat(p, suffix, suffix_len);
```

Comment:
This one might not matter, but I think it makes more sense like this anyway.

With the previous call, `strncat` is basically appending `max(len, strlen(suffix))` characters to `p`, even though `p` is only `len` bytes long. Because of the way we calculate `len`, we know that `strlen(suffix)` is less than `len` so I think there's no possibility of UB regardless of what we pass in as a length. But I think this will make Coverity more happy and it makes more sense to me personally.